src/legacy/ui/public/index_patterns/_index_pattern.d.ts

```diff
@@ -29,6 +29,10 @@ export interface IndexPattern {
   title: string;
   timeFieldName?: string;
 }
+// IndexPattern returned by IndexPatternProvider
```

Comment:
Nit: Isn't every IndexPattern instance returned by the IndexPatternProvider? In this case it should be save to add this to the regular interface

Reply:
I wanted to, but, this would break 30+ tests, so I decided to separate IndexPattern and IndexPatternEnhanced ... maybe I should drop a not there